### PR TITLE
140: address CRAN reviewer comments

### LINKED ISSUE #140: Address CRAN reviewer comments
To do:
- [x] Please use only undirected quotation marks in the description text. e.g.
`TMB` --> 'TMB'
- [x] If there are references describing the methods in your package, please
add these in the description field of your DESCRIPTION file in the form
authors (year) <doi:...>
authors (year) <arXiv:...>
authors (year, ISBN:...)
or if those are not available: <https:...>
with no space after 'doi:', 'arXiv:', 'https:' and angle brackets for
auto-linking.
(If you want to add a title as well please put it in quotes: "Title")
- [x] Please add \value to .Rd files regarding exported methods and explain
the functions results in the documentation. Please write about the
structure of the output (class) and also what the output means. (If a
function does not return a value, please document that too, e.g.
\value{No return value, called for side effects} or similar)
Missing Rd-tags:
  - [x] component.Rd: \value
  - [x] free_cores.Rd: \arguments --> does not have any arguments
  - [x] mmrm_methods.Rd: \value
  - [x] mmrm_tmb_methods.Rd: \value
- [x] The LICENSE file is only needed if you have additional restrictions to
the license which you have not? In that case omit the file and its
reference in the DESCRIPTION file.

### PR DESCRIPTION
**Pull request by danielinteractive**
closes #140

### CODE REVIEW

**Review by BrianLang — APPROVED** [review]
Nice and compact changes besides linter and spelling issues. Are the others on the issue checklist done in a different branch/PR?

**Comment by danielinteractive** [comment]
@BrianLang thanks - so the license file comment is a bit touching the copyright holder topic.
Based on https://r-pkgs.org/license.html#copyright-holder I would suggest that we
- remove the LICENSE file which essentially just refers to the license we anyway have (therefore we got the CRAN review comment) and Roche as copyright holder
- add all companies from authors as multiple copyright holders in the DESCRIPTION file: 
	- Roche
	- MSD @BrianLang 
	- Gilead @ywang-gilead 
	- Boehringer Ingelheim @kkmann 
Can you please confirm that this is good for you? See my new commit coming in and make corrections there regarding company name. 
Thanks a lot!!

**Comment by BrianLang** [comment]
> @BrianLang thanks - so the license file comment is a bit touching the copyright holder topic. Based on https://r-pkgs.org/license.html#copyright-holder I would suggest that we
> 
> * remove the LICENSE file which essentially just refers to the license we anyway have (therefore we got the CRAN review comment) and Roche as copyright holder
> * add all companies from authors as multiple copyright holders in the DESCRIPTION file:
>   
>   * Roche
>   * MSD @BrianLang
>   * Gilead @ywang-gilead
>   * Boehringer Ingelheim @kkmann
>     Can you please confirm that this is good for you? See my new commit coming in and make corrections there regarding company name.
>     Thanks a lot!!

Sounds good to me, thanks @danielinteractive.

File: R/tmb-methods.R [hunk 1]
```diff
@@ -103,6 +104,7 @@ vcov.mmrm_tmb <- function(object, complete = TRUE, ...) {
 #' # Variance-covariance matrix estimate for residuals:
 #' VarCorr(object)
 VarCorr.mmrm_tmb <- function(x, sigma = NA, ...) { # nolint
+  # nolint
```
Comment (BrianLang) [inline R/tmb-methods.R:107]: might not need it here if we have it above on 105, right?

**Review by BrianLang — APPROVED** [review]
Thanks @danielinteractive!

File: R/tmb-methods.R (hunk 1 above)
Comment (danielinteractive) [reply R/tmb-methods.R:107]: thx!

**Review by kkmann — APPROVED** [review]
@danielinteractive Fine by me/us. It's a bit confusing since the Apache 2.0 asks you to distribute the license file with the code but then CRAN complains.

**Comment by danielinteractive** [comment]
So I got the ok here too. @ywang-gilead if you approve I can merge :-)

**Review by ywang-gilead — APPROVED** [review]

**Pull request merged**